### DISCUSSION TITLE
why get_rotation_matrix_from_axis_angle only need one parameter?

### DISCUSSION THREAD

**Discussion by Carloszone** [isl-org/Open3D#6539] [Q&A] [answered]
according to the doc, get_rotation_matrix_from_axis_angle only needs one (3, 1)  parameter rotation.
"open3d.geometry.get_rotation_matrix_from_axis_angle(rotation: numpy.ndarray[numpy.float64[3, 1]]) → numpy.ndarray[numpy.float64[3, 3]]"

But based on its name, I think it needs two parameters to get the rotation matrix:  roatation_axis and rotation_angle

I tried to pass two parameters and got an error:
"
   rotation_matrix = o3d.geometry.get_rotation_matrix_from_axis_angle(rotation_axis.reshape(3, 1), rotation_angle)
TypeError: get_rotation_matrix_from_axis_angle(): incompatible function arguments. The following argument types are supported:
    1. (rotation: numpy.ndarray[numpy.float64[3, 1]]) -> numpy.ndarray[numpy.float64[3, 3]]
"

I still don't know how this method works

**Comment by saurabheights** [comment, marked as answer]
@Carloszone See the example here https://en.wikipedia.org/wiki/Axis%E2%80%93angle_representation#Example. The rotation angle can be stored in rotation vector as its magnitude.
